test/controllers/IfsControllerSpec.scala

```diff
@@ -73,64 +84,68 @@ class IfsControllerSpec extends SpecBase{
       eventTimestamp = OffsetDateTime.now(),
       feedbackId,
       metaData = List(
-        Map("nino" -> "nino"),
-        Map("taxYear" -> "2023"),
+        Map("nino"    -> "nino"),
+        Map("taxYear" -> "2023")
       ),
-      payload = Some(Messages(Some(List(IFRequestPayload(
-        messageId = "messageId",
-        englishAction = IFRequestPayloadAction(
-          title = "title",
-          message = "message",
-          action = "action",
-          path = "path",
-          links = None
-        ),
-        welshAction = IFRequestPayloadAction(
-          title = "TODO - title",
-          message = "TODO - message",
-          action = "TODO - action",
-          path = "TODO - path",
-          links = None
-        ))))
-      )))
-
-    val fakeRequest: FakeRequest[JsValue] = FakeRequest("POST", "/interaction-data/store-interactions").withBody(Json.toJson(req)).withHeaders(("Authorization", "ABCD1234"),("Content-Type", "application/json") )
+      payload = Some(
+        Messages(Some(List(IFRequestPayload(
+          messageId = "messageId",
+          englishAction = IFRequestPayloadAction(
+            title = "title",
+            message = "message",
+            action = "action",
+            path = "path",
+            links = None
+          ),
+          welshAction = IFRequestPayloadAction(
+            title = "TODO - title",
+            message = "TODO - message",
+            action = "TODO - action",
+            path = "TODO - path",
+            links = None
+          )
+        )))))
+    )
+
+    val fakeRequest: FakeRequest[JsValue] = FakeRequest("POST", "/interaction-data/store-interactions")
+      .withBody(Json.toJson(req))
+      .withHeaders(("Authorization", "ABCD1234"), ("Content-Type", "application/json"))
     controller.submit().apply(fakeRequest)
   }
 
   "IfsController.submit()" when {
 
     "generate report: provided with a calculation id in metadata" must {
       "return 204" in {
-        val result = submitStoreInteraction("good one")
+        val result = submitStoreInteraction("good one", None)
         status(result) must be(NO_CONTENT)
       }
     }
 
-    "generate report: provided with a calculation id in metadata to trigger invalid correlationId" must {
```

Comment:
is this test removed?

Reply:
While working on a new test case, I accidentally removed a test method. It's available now with latest commit.